src/js/modules/infragistics.ui.videoplayer.js

```diff
@@ -49,223 +49,1826 @@
+				/* type="string" Get or set the banner image url.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+						banners: [{
+							imageUrl: "../Content/images/Banner1.png",
+							link: "http://quince.infragistics.com/",
+							times: [5, 20, 60],
+							visible: false,
+							closeBanner: true,
+							animate: true,
+							autohide: true,
+							hidedelay: 10000,
+							width: "200px",
+							height: "67px"
+						}]
+					});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var imageUrl = banners[0].imageUrl;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].imageUrl = "../Content/images/Banner1.png";
+				```
+				*/
 				imageUrl: "",
-				/* type="array" Get or set an array of numbers. Each number specifies on which second in the movie the banner will pop. */
+				/* type="array" Get or set an array of numbers. Each number specifies on which second in the movie the banner will pop.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var times = banners[0].times;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].times = [5, 20, 60];
+				```
+				*/
 				times: [],
-				/* type="bool" Get or set whether the user will be able to close the banner or not. */
+				/* type="bool" Get or set whether the user will be able to close the banner or not.
+				```
+					//Initialize
+					$(".selector").igVideoPlayer({
+							banners: [{
+								imageUrl: "../Content/images/Banner1.png",
+								link: "http://quince.infragistics.com/",
+								times: [5, 20, 60],
+								visible: false,
+								closeBanner: true,
+								animate: true,
+								autohide: true,
+								hidedelay: 10000,
+								width: "200px",
+								height: "67px"
+							}]
+						});
+
+					//Get
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					var closeBanner = banners[0].closeBanner;
+
+					//Set
+					var banners = $(".selector").igVideoPlayer("option", "banners");
+					banners[0].closeBanner = true;
+				```
+				*/
 				closeBanner: true,
-				/* type="bool" Get or set whether to apply animation effects when showing or hiding the banner. If set to true, the animation is played for banner.duration in milliseconds. */
+				/* type="bool" Get or set whether to apply animation effects when showing or hiding the banner. If set to true, the animation is played for banner.duration in milliseconds.
```

Comment:
banner.duration should be an option link.

`[duration](ui.igvideoplayer#options:banners.duration)`